newton/_src/sim/builder.py

```
@@ -5089,13 +5110,15 @@ def add_muscle(
 
     def add_shape(
         self,
+        *,
         body: int,
         type: int,
```

Ruff (0.15.6)

[error] Function argument `type` is shadowing a Python builtin

(A002)